nixos/lib/test-driver/test-driver.py

```diff
@@ -710,6 +711,13 @@ def unblock(self):
         """
         self.send_monitor_command("set_link virtio-net-pci.1 on")
 
+    def copy_file_from_host(self, from_path, to_path):
```

Comment:
This probably needs type annotations and a rebase, as the test driver got annotations too.